pymc/distributions/continuous.py

```diff
@@ -1908,6 +1910,136 @@ def icdf(value, nu, mu, sigma):
         )
 
 
+class SkewStudentTRV(RandomVariable):
+    name = "skewstudentt"
+    ndim_supp = 0
+    ndims_params = [0, 0, 0, 0]
+    dtype = "floatX"
+    _print_name = ("SkewStudentT", "\\operatorname{SkewStudentT}")
+
+    @classmethod
+    def rng_fn(cls, rng, a, b, mu, sigma, size=None) -> np.ndarray:
+        return np.asarray(stats.jf_skew_t.rvs(a=a, b=b, size=size, random_state=rng)) * sigma + mu
```

Comment:
Can't sigma and mu be passed as arguments to the rvs? 

The way you wrote it has a similar bug that was addressed by this recent bugfix, when there are batched sigma and mu and size is None: https://github.com/pymc-devs/pymc/pull/7288

Reply:
You marked as resolved but I still see the old code?